recipes/perl-complete-common/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "perl-complete-common" %}
+{% set version = "0.22" %}
+{% set sha256 = "6293797af742ec6bca02d8aaa4181b1eecbec5ef3cfbb0b6acd4cd9a3366b12f" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/P/PE/PERLANCAR/Complete-Common-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  host:
+    - perl
+
+  run:
+    - perl
+
+test:
+  # Perl 'use' tests
+  imports:
+    - Complete::Common
+
+  # You can also put a file called run_test.pl (or run_test.py) in the recipe
+  # that will be run at test time.
+
+about:
+  home: https://metacpan.org/release/Complete-Common
+  license: GPL-1.0-or-later
+  license_file: LICENSE.txt
```

```suggestion
  license: GPL-1.0-or-later OR Artistic-1.0-Perl
  license_file: LICENSE
```
The packages already contains a `LICENSE` file which we can reuse. Please also remove the `LICENSE.txt` from the recipe directory.